oak/server/grpc_stream.cc

```diff
@@ -16,8 +16,36 @@
 
 #include "oak/server/grpc_stream.h"
 
+#include "absl/memory/memory.h"
+#include "asylo/util/logging.h"
+
 namespace oak {
 
+namespace {
+
+// Converts a gRPC ByteBuffer into a vector of bytes.
+std::unique_ptr<std::vector<uint8_t>> Unwrap(const ::grpc::ByteBuffer& buffer) {
```

Comment:
Would it make sense to just return `std::vector` (without the extra smart pointer wrapping)?

Reply:
Yes, that would be better.